### PR TITLE
feat(core): Add license:info command

### PR DESCRIPTION
**Pull request by csuermann**
This PR adds a `license:info` command that prints out information about the instance's license certificate (see below)

```
2023-04-21T07:43:14.854Z | info     | Printing license information:
## CONSUMER CONFIG ##
tenantId: 1001
productIdentifier: n8n-0.225.0
deviceFingerprint: 1c80059f3d82d22a985f35324da517592ba3e58152d5d36a0fe577593fd069f0
--
## LICENSE CERT ##
version: 2
tenantId: 1 
consumerId: e49c5cc0-a804-4e7d-9e9b-35df541ca172
deviceFingerprint: 6d80059f3d82d22a985f36384da517592ba3e48252d5d36a0fe577595fd069f0 
createdAt: Thu Apr 20 2023 23:40:55 GMT+0200 (Central European Summer Time)
issuedAt: Thu Apr 20 2023 23:40:55 GMT+0200 (Central European Summer Time)
expiresAt: Sun Apr 30 2023 23:40:55 GMT+0200 (Central European Summer Time)
terminatesAt: Mon Apr 13 2048 23:40:55 GMT+0200 (Central European Summer Time)
isEphemeral: false
isValid: true
isRenewalDue: false
entitlements: 1
--
## ENTITLEMENT 1 ##
id: 2ac7cc57-3c53-4dca-b593-bc56c68c5efb
productId: 239e5ce0-c380-4f0c-8dbc-35280d6d8a64
validFrom: Thu Apr 20 2023 23:40:55 GMT+0200 (Central European Summer Time)
validTo: Thu Apr 20 2023 23:42:55 GMT+0200 (Central European Summer Time)
features: {"planName":"Enterprise","feat:ldap":true,"quota:users":-1,"feat:sharing":true,"feat:logStreaming":true,"quota:activeWorkflows":-1}
featureOverrides: {"quota:activeWorkflows":50}
```

### CODE REVIEW

File: packages/cli/src/commands/start.ts [hunk 1]
```diff
@@ -188,12 +188,19 @@ export class Start extends BaseCommand {
 		const license = Container.get(License);
 		await license.init(this.instanceId);
 
+		const hasCert = (await license.loadCertStr()).length > 0;
```
Comment (netroy) [inline packages/cli/src/commands/start.ts:191]: These changes look unrelated to the PR title. can we create a separate PR for this change please 🙏🏽

**Review by netroy — COMMENTED** [review]
besides the one comment, LGTM

**Review by Joffcom — APPROVED** [review]
👍🏻

**Comment by janober** [comment]
Got released with [`n8n@0.226.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.226.1)
